manifests/basic_ucp/ucp-armada.yaml

```diff
@@ -0,0 +1,505 @@
+          pipeline = cors sizelimit url_normalize request_id admin_token_auth build_auth_context token_auth json_body ec2_extension_v3 s3_extension service_v3
+
+          [app:public_version_service]
+          use = egg:keystone#public_version_service
+
+          [app:admin_version_service]
+          use = egg:keystone#admin_version_service
+
+          [pipeline:public_version_api]
+          pipeline = cors sizelimit url_normalize public_version_service
+
+          [pipeline:admin_version_api]
+          pipeline = cors sizelimit url_normalize admin_version_service
+
+          [composite:main]
+          use = egg:Paste#urlmap
+          /v2.0 = public_api
+          /v3 = api_v3
+          / = public_version_api
+
+          [composite:admin]
+          use = egg:Paste#urlmap
+          /v2.0 = admin_api
+          /v3 = api_v3
+          / = admin_version_api
+    replicas: 2
+    images:
+      db_sync: kolla/ubuntu-source-keystone:2.0.2
+      api: kolla/ubuntu-source-keystone:2.0.2
+    labels:
+      node_selector_key: ucp-control-plane
+      node_selector_value: enabled
+  source:
+    type: git
+    location: git://git.openstack.org/openstack/openstack-helm
+    subpath: keystone
+  dependencies:
+    - helm-toolkit
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: maas-postgresql
+data:
+  chart_name: maas-postgresql
+  release: maas-postgresql
+  namespace: maas
+  install:
+    no_hooks: false
+  upgrade:
+    no_hooks: false
+  pre:
+    delete: []
+    create: []
+  post:
+    delete: []
+    create: []
+  values:
+    development:
+      enabled: false
+    labels:
+      node_selector_key: ucp-control-plane
+      node_selector_value: enabled
+  source:
+    type: git
+    location: git://git.openstack.org/openstack/openstack-helm-addons
+    subpath: postgresql
+  dependencies: []
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: maas
+data:
+  chart_name: maas
+  release: maas
+  namespace: maas
+  install:
+    no_hooks: false
+  upgrade:
+    no_hooks: false
+  pre:
+    delete: []
+    create: []
+  post:
+    delete: []
+    create: []
+  values:
+    bootdata_url: http://172.20.0.12:31000/api/v1.0/bootdata/
+    credentials:
+      secret:
+        namespace: drydock
+    images:
+      maas_rack: quay.io/attcomdev/maas-rack:2.2.0-no-ipmi-patch
+    labels:
+      node_selector_key: ucp-control-plane
+      node_selector_value: enabled
+    network:
+      proxy:
+        node_port:
+          enabled: true
+          port: 31800
+      gui:
+        node_port:
+          enabled: true
+          port: 31900
+    conf:
+      maas:
+        url: http://172.20.0.12:31900/MAAS
+        proxy:
+          enabled: 'false'
+          server: http://one.proxy.att.com:8080
+        ntp:
+          servers: ntp.ubuntu.com
+        dns:
+          upstream_servers: 8.8.8.8
+      secret: 3858a12230ac3c915f300c664f12063f
+  source:
+    type: local
```

Comment:
`local` won't work for others and likely is not desired as it means the promenade instructions are to both push up the bundle as well as _some_ chart sources as well as.  I am also not sure without looking whether the armada loader in promenade even mounts this directory into the pod even if I did have the placed here on the physical genesis host.